### PR TITLE
Updating office scripts packages versions

### PR DESCRIPTION
**Pull request by igor-ribeiiro**
Updated all office adding scripts to their latest versions and fixed the functions necessary changes.

Tested with:
```npm run test```
And all tests gave the exact same results as before making these changes

### CODE REVIEW

**Comment by akrantz** [comment]
@TCourtneyOwen Shouldn't we merge and publish this PR first?
https://github.com/OfficeDev/Office-Addin-Scripts/pull/382

File: test/ui-test.ts [hunk 1]
```diff
@@ -27,8 +27,8 @@ hosts.forEach(function (host) {
             // Call startDebugging to start dev-server and sideload
             const devServerCmd = `npm run dev-server -- --config ./test/webpack.config.js `;
             const devServerPort = parseNumber(process.env.npm_package_config_dev_server_port || 3000);
-            await startDebugging(manifestPath, AppType.Desktop, toOfficeApp(host), undefined, undefined, 
-                devServerCmd, devServerPort, undefined, undefined, undefined, false /* enableDebugging */);
+            await startDebugging(manifestPath, {  appType:AppType.Desktop, app:toOfficeApp(host), devServerCommandLine:devServerCmd,
```
Comment (akrantz) [inline test/ui-test.ts:30]: @igor-ribeiiro I'm not sure that you created your branch off of master. These changes look like they are from Courtney's changes. Let's chat offline and I can explain how to fix this.

**Review by akrantz — CHANGES_REQUESTED** [review]
See comments...

**Comment by TCourtneyOwen** [comment]
Yes I agree we should merge and publish OfficeDev/Office-Addin-Scripts#382 but we need to see why the CI loop failed for that PR

**Comment by TCourtneyOwen** [comment]
Latest versions of office-addin-scripts packages have been published so all the versions in package.json in this PR should be bumped up one for the patch version

File: test/ui-test.ts (hunk 1 above)
Comment (akrantz) [reply test/ui-test.ts]: With the update of the office-addin-debugging package, the startDebugging() function changed. I pushed a commit which has the same changes but cleans up the formatting.

Comment (igor-ribeiiro) [reply test/ui-test.ts]: Just looked into the changes you committed and they look fine to me

**Comment by TCourtneyOwen** [comment]
Why are there changes to so many other files in this PR if this was only intended to update packages? Was there perhaps a merge issue here from master?

**Comment by akrantz** [comment]
I had started doing this in a previous PR, and along the way I found that there were lint errors and things to be fixed. I merged this into Igor's PR so he could finish it all.

**Review by TCourtneyOwen — APPROVED** [review]

**Review by akrantz — APPROVED** [review]

**Comment by akrantz** [comment]
There was also the changes needed for startDebugging() breaking change.

**Pull request merged**